### PR TITLE
Add labels template with sync

### PR DESCRIPTION
**Pull request by wopox1337**
- close https://github.com/rehlds/.github/issues/17

### CODE REVIEW

**Comment by wopox1337** [comment]
@stamepicmorg Please review and approve the following list of labels.

> [!WARNING]
> Labels that do not match the list [are **deleted**](https://github.com/rehlds/.github/actions/runs/11650010720/job/32438238836#step:4:36) during synchronization (but, existing matching labels are not touched). Accordingly, all labels in regamedll, rehlds, metamod-r will be deleted and will need to be reassigned.

File: .github/labels.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+- name: "🔥 priority: high"
+  description: "Tasks with high priority that require immediate attention."
+  color: "FF5733"
+- name: "⚠️ priority: medium"
+  description: "Medium priority tasks that should be addressed soon."
+  color: "FFC300"
+- name: "🕒 priority: low"
+  description: "Low priority tasks that can be postponed for the future."
+  color: "C4C4C4"
+- name: "🐞 bug"
```
Comment (stamepicmorg) [inline .github/labels.yml:10]:
```suggestion
- name: "Type: 🐞 bug"
```

File: .github/labels.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  description: "An error that needs fixing."
+  color: "FF5733"
+- name: "⚡ performance"
```
Comment (stamepicmorg) [inline .github/labels.yml:13]:
```suggestion
- name: "Type:⚡ performance"
```

File: .github/labels.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Optimization of system or code performance."
+  color: "FFC107"
+- name: "🔒 security"
```
Comment (stamepicmorg) [inline .github/labels.yml:16]:
```suggestion
- name: "Type: 🔒 security"
```

File: .github/labels.yml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Vulnerabilities or issues related to security."
+  color: "DC3545"
+- name: "✂️ duplicate"
```
Comment (stamepicmorg) [inline .github/labels.yml:19]:
```suggestion
- name: "Resolution: ✂️ duplicate"
```

File: .github/labels.yml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Duplicate of an existing task or issue."
+  color: "FF5733"
+- name: "🚫 wontfix"
```
Comment (stamepicmorg) [inline .github/labels.yml:22]:
```suggestion
- name: "Resolution: 🚫 wontfix"
```

File: .github/labels.yml [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+  description: "The issue will not be fixed; the task is closed."
+  color: "DC3545"
+- name: "❓ invalid"
```
Comment (stamepicmorg) [inline .github/labels.yml:25]:
```suggestion
- name: "Resolution: ❓ invalid"
```

File: .github/labels.yml [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Invalid task, lacking sufficient information or incorrect format."
+  color: "C4C4C4"
+- name: "🚀 enhancement"
```
Comment (stamepicmorg) [inline .github/labels.yml:28]:
```suggestion
- name: "Type: 🚀 enhancement"
```

File: .github/labels.yml [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Improvement or addition of a new feature."
+  color: "28A745"
+- name: "💡 feature request"
```
Comment (stamepicmorg) [inline .github/labels.yml:31]:
```suggestion
- name: "Type: 💡 feature request"
```

File: .github/labels.yml [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Request for a new feature or enhancement."
+  color: "FF851B"
+- name: "📚 documentation"
```
Comment (stamepicmorg) [inline .github/labels.yml:34]:
```suggestion
- name: "Type: 📚 documentation"
```

File: .github/labels.yml [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Tasks related to updating or improving documentation."
+  color: "007BFF"
+- name: "🛠️ refactor"
```
Comment (stamepicmorg) [inline .github/labels.yml:37]:
```suggestion
- name: "Type: 🛠️ refactor"
```

File: .github/labels.yml [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Refactoring code without adding new functionality."
+  color: "6F42C1"
+- name: "✅ test"
```
Comment (stamepicmorg) [inline .github/labels.yml:40]:
```suggestion
- name: "Type: ✅ test"
```

File: .github/labels.yml [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Related to testing and verification of code."
+  color: "20C997"
+- name: "🚧 in progress"
```
Comment (stamepicmorg) [inline .github/labels.yml:43]:
```suggestion
- name: "Status: 🚧 in progress"
```

File: .github/labels.yml [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Task currently being worked on."
+  color: "FFC300"
+- name: "👀 needs review"
```
Comment (stamepicmorg) [inline .github/labels.yml:46]:
```suggestion
- name: "Status: 👀 needs review"
```

File: .github/labels.yml [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Requires code or changes review."
+  color: "17A2B8"
+- name: "🗣️ needs feedback"
```
Comment (stamepicmorg) [inline .github/labels.yml:49]:
```suggestion
- name: "Status: 🗣️ needs feedback"
```

File: .github/labels.yml [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Comments or feedback needed from the community or team."
+  color: "6C757D"
+- name: "🕹️ on hold"
```
Comment (stamepicmorg) [inline .github/labels.yml:52]:
```suggestion
- name: "Status: 🕹️ on hold"
```

File: .github/labels.yml [hunk 16]
```diff
@@ -0,0 +1,75 @@
+- name: "🔥 priority: high"
```
Comment (stamepicmorg) [inline .github/labels.yml:1]:
```suggestion
- name: "Priority: 🔥 high"
```

File: .github/labels.yml [hunk 17]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Tasks with high priority that require immediate attention."
+  color: "FF5733"
+- name: "⚠️ priority: medium"
```
Comment (stamepicmorg) [inline .github/labels.yml:4]:
```suggestion
- name: "Priority: ⚠️ medium"
```

File: .github/labels.yml [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Medium priority tasks that should be addressed soon."
+  color: "FFC300"
+- name: "🕒 priority: low"
```
Comment (stamepicmorg) [inline .github/labels.yml:7]:
```suggestion
- name: "Priority: 🕒 low"
```

File: .github/labels.yml [hunk 19]
```diff
@@ -0,0 +1,75 @@
...
+- name: "👀 needs review"
+  description: "Requires code or changes review."
+  color: "17A2B8"
+- name: "🗣️ needs feedback"
+  description: "Comments or feedback needed from the community or team."
+  color: "6C757D"
+- name: "🕹️ on hold"
+  description: "Task temporarily on hold."
+  color: "C4C4C4"
+- name: "🟢 easy"
```
Comment (stamepicmorg) [inline .github/labels.yml:55]:
```suggestion
- name: "Difficulty:  🟢 easy"
```

File: .github/labels.yml [hunk 20]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Easy task that can be completed quickly."
+  color: "28A745"
+- name: "🟡 medium"
```
Comment (stamepicmorg) [inline .github/labels.yml:58]:
```suggestion
- name: "Difficulty:  🟡 medium"
```

File: .github/labels.yml [hunk 21]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Task of medium difficulty requiring moderate effort."
+  color: "FFC300"
+- name: "🔴 hard"
```
Comment (stamepicmorg) [inline .github/labels.yml:61]:
```suggestion
- name: "Difficulty:  🔴 hard"
```

File: .github/labels.yml [hunk 22]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Difficult task requiring significant resources and time."
+  color: "DC3545"
+- name: "🧹 chore"
```
Comment (stamepicmorg) [inline .github/labels.yml:64]:
```suggestion
- name: "Difficulty: 🧹 chore"
```

File: .github/labels.yml [hunk 23]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Routine tasks, maintenance, or non-user-facing changes."
+  color: "FAD8C7"
+- name: "🛠️ maintenance"
```
Comment (stamepicmorg) [inline .github/labels.yml:67]:
```suggestion
- name: "Type: 🛠️ maintenance"
```

File: .github/labels.yml [hunk 24]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Tasks related to technical debt, infrastructure upkeep."
+  color: "FFD700"
+- name: "⚙️ infrastructure"
```
Comment (stamepicmorg) [inline .github/labels.yml:70]:
```suggestion
- name: "Type: ⚙️ infrastructure"
```

File: .github/labels.yml [hunk 25]
```diff
@@ -0,0 +1,75 @@
...
+  description: "Tasks for maintaining or improving system infrastructure."
+  color: "B0C4DE"
+- name: "⚠️ triage"
```
Comment (stamepicmorg) [inline .github/labels.yml:73]:
```suggestion
- name: "Status: ⚠️ triage"
```

File: .github/labels.yml [hunk 26]
```diff
@@ -0,0 +1,75 @@
...
+  description: "An error that needs fixing."
+  color: "FF5733"
+- name: "⚡ performance"
+  description: "Optimization of system or code performance."
+  color: "FFC107"
+- name: "🔒 security"
+  description: "Vulnerabilities or issues related to security."
+  color: "DC3545"
+- name: "✂️ duplicate"
+  description: "Duplicate of an existing task or issue."
```
Comment (stamepicmorg) [inline .github/labels.yml:20]:
```suggestion
  description: "Duplicate of an existing task or issue."
  - name: "Resolution: ❇️ Resolved"
  description: "The issue resolved, but not fully closed."
  - name: "Status: ✅ Done"
  description: "The issue done and closed."
```

**Review by stamepicmorg — COMMENTED** [review]
Awesome! Thank you!

**Pull request merged**